pkg/build/builder/sti.go

```diff
@@ -265,8 +267,8 @@ func (s *S2IBuilder) Build() error {
 		} else {
 			glog.V(2).Infof("No push secret provided")
 		}
-		glog.V(1).Infof("Pushing %s image ...", pushTag)
-		if err := pushImage(s.dockerClient, pushTag, pushAuthConfig); err != nil {
+		fmt.Fprintf(s.out, "Pushing %s image ...\n", pushTag)
```

Comment:
because I want it to be printed regardless of the log level, given that image push progress is going to be written to the same output stream.